requirements/edx/local.txt

```diff
@@ -1,12 +1,13 @@
 # Python libraries to install that are local to the edx-platform repo
 -e .
--e common/lib/calc
+
 -e common/lib/capa
--e common/lib/chem
--e common/lib/dogstats
--e common/lib/safe_lxml
--e common/lib/sandbox-packages
--e common/lib/symmath
 -e common/lib/xmodule
 
+-e openedx/core/lib/calc
```

Comment:
Code in openedx is already on the python path.  Having code available via two separate import paths already causes problems in lms.  My 2 cents is that if the code needs to be pip installed separately from edx-platform, it isn't ready to be moved out of common.